packages/kit/src/exports/index.js

```diff
@@ -110,6 +110,18 @@ export function redirect(status, location) {
 	);
 }
 
+export const acceptResponse = new Response(null, {
```

Comment:
I don't think is used anywhere else so we can avoid exporting it to prevent it from being accessed by users.
```suggestion
const acceptResponse = new Response(null, {
```